src/client/sky.cpp

```diff
@@ -766,3 +624,87 @@ void Sky::update(float time_of_day, float time_brightness,
 			video::SColorf(pointcolor), m_horizon_blend() * 0.25);
 	}
 }
+
+void Sky::draw_sun(video::IVideoDriver *driver, float sunsize, video::SColor suncolor, video::SColor suncolor2, float wicked_time_of_day) {
+	static const u16 indices[4] = {0, 1, 2, 3};
+	std::array<video::S3DVertex, 4> vertices;
+	if (!m_sun_texture) {
+		driver->setMaterial(m_materials[1]);
+		const std::array<float, 4> sunsizes = {sunsize * 1.7f, sunsize * 1.2f, sunsize, sunsize * 0.7f};
```

Comment:
this could be a plain old C-array, is there any reason this is an `std::array`?

Reply:
Yes it could. But `std::array` are nicer, and it's easier to pass to functions.If it creates to much inconsistency with the codebase, I can use C-style arrays

Reply:
But this array isn't being passed anywhere, hence my question.
I prefer C-style arrays for `sunsizes`, `moonsizes_1`, `moonsizes_2`.

Reply:
You're right, I just used them anywhere I added arrays. I agree in this case it makes sense to use, C-style arrays, I'll fix it.

Reply:
Done.